internal/pkg/keybindings/listhandlers.go

```diff
@@ -722,3 +722,48 @@ func (h *CommandPanelAzureSearchQueryHandler) CommandPanelNotification(state vie
 }
 
 ////////////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////////////
+type ListCopyItemIDHandler struct {
+	ListHandler
+	List      *views.ListWidget
+	StatusBar *views.StatusbarWidget
+}
+
+var _ Command = &ListCopyItemIDHandler{}
+
+func NewListCopyItemIDHandler(list *views.ListWidget, statusBar *views.StatusbarWidget) *ListCopyItemIDHandler {
+	handler := &ListCopyItemIDHandler{
+		List:      list,
+		StatusBar: statusBar,
+	}
+	handler.id = HandlerIDListCopyItemID
+	return handler
+}
+
+func (h ListCopyItemIDHandler) Fn() func(g *gocui.Gui, v *gocui.View) error {
+	return func(g *gocui.Gui, v *gocui.View) error {
+		return h.Invoke()
+	}
+}
+
+func (h *ListCopyItemIDHandler) DisplayText() string {
+	return "Copy current item ID"
```

Comment:
nit: I think "Copy current resource ID" is more consistent with Azure terminology.